### ISSUE TITLE
Node-redis container images should be build with Actions

### ISSUE THREAD

**Issue by kelson42** [openzim/mwoffliner#1815]
Currently it has to be pushed manually

**Comment by kelson42** [comment]
@benoit74 AFAIK this is not the case anymore (see https://github.com/openzim/mwoffliner/blob/main/docker/Dockerfile) and therefore this issue is outdated?!

**Comment by benoit74** [comment]
Indeed, we do not use this intermediate image anymore, we have one single image / Dockerfile.
